packages/cli/tsconfig.json

```diff
@@ -2,7 +2,8 @@
   "extends": "../typescript/tsconfig.library.json",
   "compilerOptions": {
     "rootDir": "src",
-    "outDir": "lib"
+    "outDir": "lib",
+    "esModuleInterop": true
```

Comment:
again, looks weird to me that we need this, when we have this setting already in tsconfig.library.json